### PR TITLE
LGA 2688 Ensure MAX_RETRIES is typecast to an integer

### PR DESCRIPTION
**Pull request by BenMillar-MOJ**
## What does this pull request do?

- Ensures the environment variable `MAX_RETRIES` is typecast to an integer.
- Handles TypeErrors if that typecast cannot be performed.

## Any other changes that would benefit highlighting?

Intentionally left blank.

## Checklist

- [x] Provided JIRA ticket number in the title, e.g. "LGA-152: Sample title"

### CODE REVIEW

File: config/base_config.py [hunk 1]
```diff
@@ -42,6 +42,10 @@ class BaseConfig:
 
     # Limits the number of retries, 32 Retries means the email will be lost if after 24 hours we are still unable to get a response
     MAX_RETRIES = os.environ.get("MAX_RETRIES", 32)
+    try:
+        MAX_RETRIES = int(MAX_RETRIES)
+    except TypeError as e:
+        raise TypeError(f"MAX_RETRIES could not be interepreted as an integer, {e}")
```
Comment (sarahyjja) [inline config/base_config.py]: Just a little typo ;) `interpreter`. Otherwise, it's great!

Comment (BenMillar-MOJ) [reply config/base_config.py]: Argh! 
Thanks for spotting Sarah, appreciate it! 😄

**Review by sarahyjja — APPROVED** [review]
LGTM 🎈

**Review by psweeting1 — APPROVED** [review]
LGTM

**Pull request merged**